desc/objectives/_coils.py

```diff
@@ -753,13 +753,28 @@ def build(self, use_jit=True, verbose=1):
             Bplasma = compute_B_plasma(
                 eq, eval_grid, self._source_grid, normal_only=True
             )
+        field = self._field
+        if hasattr(field, "Phi_mn"):
```

Comment:
would it be better to get this all to work with the existing `desc.compute.utils.get_transforms` helper? That would work for coils etc as well.